src/utils/utils.js

```diff
@@ -205,6 +205,14 @@ export function getInfomediaDetails(article) {
   return details;
 }
 
+/**
+ * Get agency parameters from vip core - check if settings allows borrowercheck
+ * for given branch/agency.
+ * @param agencyId
```

Comment:
måske kald parameteren  "libraryId" og så skriv kommentar "can either be agency- or branchId depending on if FFU/foreign library or 'folkebibliotek'"

Det vil være mere tydligt.